server/router/upload.ts

```diff
@@ -0,0 +1,50 @@
+import express from 'express';
+import * as AWS from 'aws-sdk';
+import { Condition } from 'dynamoose/dist/Condition';
+import * as db from './common';
+import { Rider } from '../models/rider';
+import { Driver } from '../models/driver';
+import { Dispatcher } from '../models/dispatcher';
+
+const router = express.Router();
+
+const BUCKET_NAME = 'dti-carriage-staging-public';
```

Comment:
Does it matter if this is publicly here or are there some security measures in place on the AWS side? Should we be putting this in a .env?

Reply:
It shouldn't matter since it is secured from the AWS side. In this case I think it was set to have public access - I can't view or edit the specific policy but the dev leads could if we want to make it more secure